build.gradle

```diff
@@ -52,26 +45,21 @@ buildscript {
                 'targetSdk' : 30
```

Comment:
Looks like we need to bump the `compileSdk` to 33 for some of the dependencies. Did you give this a try?

Reply:
Yes but several other things go wrong after that.